### PR TITLE
Prepare for 0.0.17-dev-01 release.

### PR DESCRIPTION
**Pull request by kenzieschmoll**

### CODE REVIEW

File: packages/devtools/lib/devtools.dart [hunk 1]
```diff
@@ -5,4 +5,4 @@
 /// The DevTools application version.
 // Note: when updating this, please update the corresponding version in the
 // pubspec.
-const String version = '0.0.17-dev';
+const String version = '0.0.17-dev-01';
```
Comment (jacob314) [inline packages/devtools/lib/devtools.dart]: should be dev.1
not dev-01

Comment (jacob314) [reply packages/devtools/lib/devtools.dart]: same elsewhere

File: packages/devtools/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 0.0.17-dev-01
```
Comment (jacob314) [inline packages/devtools/CHANGELOG.md]: Add the date to this entry.

**Comment by DanTup** [comment]
Do we mean to include the `browser_launcher` changes in this release? (If so, we need to publish a new `browser_launcher` first, then rev the version in this one).

Though FWIW, I'm working on another change for `devtools_server` for #582 I'd like to include too if we do.

**Pull request merged**